airgun/entities/rhai/plan.py

```diff
@@ -40,6 +42,36 @@ def update(self, entity_name, values):
         view = PlanEditView(self.session.browser)
         view.fill_with(values, on_change=view.save.click)
 
+    def run_playbook(self, entity_name, customize=False, customize_values=None):
```

Comment:
doc-strings are missing, just for consistency. non-blocking comment

Reply:
Fair enough, let me add them real quick here

Reply:
Docstrings added in 53fd3c7e1e57539fb7a4cfb7ff1d45d44fe63de4 , which was amended and force-pushed